partiql-eval/src/main/kotlin/org/partiql/eval/PartiQLEngineBuilder.kt

```diff
@@ -1,36 +1,11 @@
 package org.partiql.eval
 
-import org.partiql.spi.connector.ConnectorBindings
-
 class PartiQLEngineBuilder {
 
-    private var catalogs: MutableMap<String, ConnectorBindings> = mutableMapOf()
-
     /**
      * Build the builder, return an implementation of a [PartiQLEngine]
      *
      * @return
      */
-    public fun build(): PartiQLEngine = PartiQLEngineDefault(catalogs)
-
-    /**
-     * Java style method for assigning a Catalog name to [ConnectorBindings].
-     *
-     * @param catalog
-     * @param bindings
-     * @return
-     */
-    public fun addCatalog(catalog: String, bindings: ConnectorBindings): PartiQLEngineBuilder = this.apply {
-        this.catalogs[catalog] = bindings
-    }
-
-    /**
-     * Kotlin style method for assigning Catalog names to [ConnectorBindings].
-     *
-     * @param catalogs
-     * @return
-     */
-    public fun catalogs(vararg catalogs: Pair<String, ConnectorBindings>): PartiQLEngineBuilder = this.apply {
-        this.catalogs = mutableMapOf(*catalogs)
-    }
+    public fun build(): PartiQLEngine = PartiQLEngineDefault()
```

Comment:
Do we even need the Builder in this case? Could just be a:
```kotlin
companion object {
  @JvmStatic
  public fun default(): PartiQLEngine = PartiQLEngineDefault()
}
```

On the PartiQLEngine interface.

Reply:
I decided to keep the builder in this PR since those interfaces has not been finalized yet.